lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -6902,9 +6902,21 @@ void EmitLoad(
 
         // f(x) +=
     case knopCall:
-        funcInfo->AcquireLoc(lhs);
-        EmitReference(lhs, byteCodeGenerator, funcInfo);
-        EmitCall(lhs, byteCodeGenerator, funcInfo, /*fReturnValue=*/ false, /*fEvaluateComponents=*/ false);
+        if (lhs->sxCall.pnodeTarget->nop == knopImport)
```

Comment:
Similar code is there in the Emit function for case knopCall. Should we refactor to one?